src/main/resources/application-RECEIVER.properties

```diff
@@ -1,9 +1,12 @@
 # Use with Spring Profile RECEIVER "java -Dspring.profiles.active=RECEIVER"
 spring.banner.location=classpath:true-connector-banner.txt
 
-logging.level.root=ERROR
-logging.level.it.eng.idsa.businesslogic=INFO
-logging.pattern.console=%d{dd-MM-yyyy HH:mm:ss.SSS} [%thread] %-5level %logger.%M - %msg%n
+logging.config=classpath:logback-RECEIVER.xml
+
+# Choose which events to log from list bellow
+#auditableEvents=ALL,NONE,SELF_DESCRIPTION,SELF_DESCRIPTION_ALL,CONTRACT_OFFER,OFFERED_RESOURCE,REPRESENTATION,USER,EXCEPTION,CONNECTOR
+# if left empty it will default to ALL
+application.logging.auditableEvents=
```

Comment:
What happens if this string is not in property file or if it is empty, like this?